bin/ingestion.js

```diff
@@ -42,18 +45,42 @@ function queueBatch(queuePopulator, taskState, qConfig, log) {
 }
 /* eslint-enable no-param-reassign */
 
+function initIngestionReaders(ingestionPopulator, cb) {
+    // TODO: use config level updated sources list. for now stick with static
+    const ingestionSources = ingestionExtConfigs.sources;
+
+    return async.each(ingestionSources, (source, next) => {
+        ingestionPopulator.addNewLogSource(source, err => {
+            if (err) {
+                log.error('error adding ingestion source', {
+                    source: source.name,
+                    prefix: source.prefix,
+                    type: source.type,
+                    method: 'IngestionPopulator::task',
+                });
+                return next(err);
+            }
+            // TODO: config change. Should somehow depend on:
+            //       Storage Location Name + Local Zenko Bucket Name
```

Comment:
Local zenko bucket name by itself should be fine. All local bucket names are unique, otherwise cloudserver throws `BucketAlreadyOwnedByYou`